pkg/porter/create_test.go

```diff
@@ -15,11 +14,23 @@ func TestCreate(t *testing.T) {
 	err := p.Create()
 	require.NoError(t, err)
 
-	configFileExists, err := p.FileSystem.Exists(config.Name)
+	configFileExists, err := p.FileSystem.Exists("porter.yaml")
 	require.NoError(t, err)
 	assert.True(t, configFileExists)
 
-	runScriptExists, err := p.FileSystem.Exists(config.RunScript)
```

Comment:
It looks like this `runScriptExists` verification has been lost in the new test -- was this intended?

Reply:
Oops, thanks that should have been moved to TestPorter_prepareDockerFilesystem